packages/core/types/src/http/file/admin/responses.ts

```diff
@@ -16,3 +16,36 @@ export interface AdminFileListResponse {
 }
 
 export type AdminFileDeleteResponse = DeleteResponse<"file">
+
+export interface AdminUploadPreSignedUrlResponse {
```

Comment:
todo: Assuming this is the HTTP response, this needs to be snake_case to follow our convention